IdfaIOS/CustomIdfa.m

```diff
@@ -0,0 +1,37 @@
+#import <AdSupport/AdSupport.h>
+#import <AppTrackingTransparency/AppTrackingTransparency.h>
+
+//Please ensure this path points to the file 'Assets/Plugins/iOS/Amplitude/Amplitude.h'
+#import "../../Plugins/iOS/Amplitude/Amplitude.h"
+
+#import "../../Plugins/iOS/Amplitude/AmplitudeCWrapper.h"
+
+typedef NSString* (^AMPAdSupportBlock)(void);
+
+void setIdfaBlockInternal(const char* instanceName)
+{
```

Comment:
this should go after Ln 11

Reply:
The standard in our app is that Objective C++ functions (with C style signatures) have the indent: 

void func()
{
...
}

while Objective C functions do the other style:

- (void)logEvent:(NSString *)eventType {
...
}

Reply:
Let's make objc and c style to be the same.